### PR TITLE
docs(launch-autoware): add how to launch yabloc or eagleye

### PR DESCRIPTION
**Pull request by KYabuuchi**
## Description
* Added a new page: "How to guides/5. Launch Autoware".
* Organized the startup procedures for YabLoc and Eagleye and made changes to the page hierarchy.
* Provided additional details about YabLoc.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The Reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by KYabuuchi** [comment]
Hi! This PR is now ready for review.

 I would like each of you to review the following: :pray: 
@yukkysaito @mitsudome-r  :  [the hierarchy structure of the documentation pages](https://autowarefoundation.github.io/autoware-documentation/pr-411/how-to-guides/integrating-autoware/launch-autoware/). 
@rsasaki0109 : about Eagleye. Specifically, ["Localization mode"](https://autowarefoundation.github.io/autoware-documentation/pr-411/how-to-guides/integrating-autoware/launch-autoware/localization-mode/) and ["Eagleye guide"](https://autowarefoundation.github.io/autoware-documentation/pr-411/how-to-guides/integrating-autoware/launch-autoware/localization-mode/eagleye-guide/).

**Comment by yukkysaito** [comment]
@KYabuuchi Thank you for your PR. LGTM :+1:

**Comment by KYabuuchi** [comment]
@kminoda I hope you take a look at ["YabLoc"](https://autowarefoundation.github.io/autoware-documentation/pr-411/how-to-guides/integrating-autoware/launch-autoware/localization-mode/yabloc-guide/) and ["Localization mode"](https://autowarefoundation.github.io/autoware-documentation/pr-411/how-to-guides/integrating-autoware/launch-autoware/localization-mode/)  :pray:

File: docs/how-to-guides/integrating-autoware/launch-autoware/localization-mode/yabloc-guide.md
Comment (kminoda) [inline docs/how-to-guides/integrating-autoware/launch-autoware/localization-mode/yabloc-guide.md:95]:
```suggestion
- In places where there are few road surface markings, such as intersections, the estimation heavily relies on GNSS, IMU, and vehicle's wheel odometry.
```

File: docs/how-to-guides/integrating-autoware/launch-autoware/localization-mode/yabloc-guide.md
Comment (kminoda) [inline docs/how-to-guides/integrating-autoware/launch-autoware/localization-mode/yabloc-guide.md:91]: Maybe this is what you meant?
```suggestion
  - This is because running both at the same time may be computationally too expensive.
```

File: docs/how-to-guides/integrating-autoware/launch-autoware/localization-mode/yabloc-guide.md
Comment (kminoda) [inline docs/how-to-guides/integrating-autoware/launch-autoware/localization-mode/yabloc-guide.md:35]:
```suggestion
  localization_mode:=camera
```

File: docs/how-to-guides/integrating-autoware/launch-autoware/localization-mode/yabloc-guide.md
Comment (kminoda) [inline docs/how-to-guides/integrating-autoware/launch-autoware/localization-mode/yabloc-guide.md:96]:
```suggestion
- If the road boundary or road surface markings are not included in the Lanelet2, the estimation is likely to fail.
```

File: docs/how-to-guides/integrating-autoware/launch-autoware/localization-mode/yabloc-guide.md
Comment (kminoda) [inline docs/how-to-guides/integrating-autoware/launch-autoware/localization-mode/yabloc-guide.md:1]: It's rather a broad suggestion which is pretty much open to discussion, but since it seems that the contents are mostly duplicated with the Readme.md of YabLoc in Autoware Universe. I guess it's better to avoid maintaining such duplicated documents, so how about removing this document and merge it into Readme, or the other way around (Remove readme and just put a link to this document)?

File: docs/how-to-guides/integrating-autoware/launch-autoware/perception.md
Comment (kminoda) [inline docs/how-to-guides/integrating-autoware/launch-autoware/perception.md:1]: Where did it come from?

File: docs/how-to-guides/integrating-autoware/launch-autoware/localization-mode/index.md
Comment (kminoda) [inline docs/how-to-guides/integrating-autoware/launch-autoware/localization-mode/index.md:64]: I think this table can completely be removed

File: docs/how-to-guides/integrating-autoware/launch-autoware/localization-mode/index.md
Comment (kminoda) [inline docs/how-to-guides/integrating-autoware/launch-autoware/localization-mode/index.md:43]: It's just a nits so can be ignored.

IMHO it's a bit confusing to have subsubsections only for Eagleye in terms of TOC (see the figure below). Given that these two subsubsections are not that long, how about just removing these titles?
```suggestion
```

image

File: docs/how-to-guides/integrating-autoware/launch-autoware/localization-mode/index.md
Comment (kminoda) [inline docs/how-to-guides/integrating-autoware/launch-autoware/localization-mode/index.md:58]: Same as above

File: docs/how-to-guides/integrating-autoware/launch-autoware/localization-mode/yabloc-guide.md
Comment (kminoda) [inline docs/how-to-guides/integrating-autoware/launch-autoware/localization-mode/yabloc-guide.md:8]:
```suggestion
YabLoc enables users localize vehicles that are not equipped with LiDAR and in environments where point cloud maps are not available.
```

File: docs/how-to-guides/integrating-autoware/launch-autoware/localization-mode/yabloc-guide.md
Comment (kminoda) [inline docs/how-to-guides/integrating-autoware/launch-autoware/localization-mode/yabloc-guide.md:12]:
```suggestion
The diagram below illustrates the basic principle of YabLoc. It extracts road surface markings by extracting the line segments using the road area obtained from graph-based segmentation. The red line at the center-top of the diagram represents the line segments identified as road surface markings. YabLoc transforms these segments for each particle and determines the particle's weight by comparing them with the cost map generated from Lanelet2.
```

**Review by kminoda — COMMENTED** [review]
Cool!! Thank you for the document :pray: 
Maybe this comment be better to handled first https://github.com/autowarefoundation/autoware-documentation/pull/411#discussion_r1243315845

File: docs/how-to-guides/integrating-autoware/launch-autoware/perception.md
Comment (KYabuuchi) [reply docs/how-to-guides/integrating-autoware/launch-autoware/perception.md:1]: I found it from https://github.com/autowarefoundation/autoware_launch/blob/main/autoware_launch/launch/autoware.launch.xml#L39

Comment (kminoda) [reply docs/how-to-guides/integrating-autoware/launch-autoware/perception.md:1]: Ah ok you volunteered to add perception mode as well. Got it, thanks!

Comment (KYabuuchi) [reply docs/how-to-guides/integrating-autoware/launch-autoware/perception.md:1]: I intentionally added that. But honestly, I don't have much knowledge about it, so the page doesn't have much content.

File: docs/how-to-guides/integrating-autoware/launch-autoware/localization-mode/yabloc-guide.md
Comment (KYabuuchi) [reply docs/how-to-guides/integrating-autoware/launch-autoware/localization-mode/yabloc-guide.md:1]: Thank you for the suggestion. It's a great idea.
Since the "limitations" and "input/output" are already written in the README of `autoware.universe`, I will try to reconsider the content of this page to avoid duplication. :raising_hand_man:

File: docs/how-to-guides/integrating-autoware/launch-autoware/localization-mode/index.md
Comment (KYabuuchi) [reply docs/how-to-guides/integrating-autoware/launch-autoware/localization-mode/index.md]: I merged the short sub-sub-sections into sub-sections in 50ff60b

File: docs/how-to-guides/integrating-autoware/launch-autoware/localization-mode/index.md
Comment (KYabuuchi) [reply docs/how-to-guides/integrating-autoware/launch-autoware/localization-mode/index.md]: I merged the short sub-sub-sections into sub-sections in 50ff60b

**Comment by KYabuuchi** [comment]
@kminoda  I have moved the introduction of YabLoc to the README in autoware.universe. The reviews pointed out have been reflected there. https://github.com/autowarefoundation/autoware.universe/pull/4100

**Review by kminoda — APPROVED** [review]
LGTM!!

**Review by kminoda — APPROVED** [review]
I once dismissed the approval since I got confused with the localization_mode and pose_estimator_mode, but now it's clear that it's not an error (complicated though), so LGTM again!

**Comment by KYabuuchi** [comment]
@rsasaki0109 Hi. I made some minor updates to the Eagleye page, so I would appreciate a quick review to ensure everything looks good. :bow: :pray:

**Comment by rsasaki0109** [comment]
Sorry. I did not notice the notification. I will check.

**Review by rsasaki0109 — APPROVED** [review]
This PR looks very good to me.

**Pull request merged**
